doc/getting_started/up2.rst

```diff
@@ -15,7 +16,7 @@ N4200* <https://ark.intel.com/products/95592/Intel-Pentium-Processor-N4200-2M-Ca
 SoCs. Both have been confirmed to work with ACRN.
 
 Connecting to the serial port
------------------------------
+=============================
 
 The UP2 board has two serial ports: one via the M10 header and one via
```

Comment:
I'd rewrite this as:

    The UP2 board has two serial ports. Please refer to the `UP2
    specifications <http://www.up-board.org/upsquared/specifications-up2/>`__
    for more information.  We'll access the serial port
    through the I/O pins in the 40-pin HAT connector using a
    `USB TTL serial cable <http://www.ftdichip.com/Products/USBTTLSerial.htm>`__.

and then continue with "Connect Pin 6..."  and end with:

    Plug the USB TTL serial cable into your PC and use a console
    emulation tool such as minicom or putty to communicate with
    the UP2 board for debugging.